flexbridge

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Run command in environment
+
+set -e -o pipefail
+
+scriptdir="$(dirname "$0")"
+prefix=$(cd "$scriptdir/../.."; /bin/pwd)
```

Comment:
I think that better than setting prefix this way and then using it on line 24 would be to test the beginning of the pwd string. But I don't think we need to make a change about that now.